From 1e1fd9b2c030f0fb9caaf585b013f70b599dc50f Mon Sep 17 00:00:00 2001
From: Alexandr Savca <alexandrsavca89@gmail.com>
Date: Sun, 4 Oct 2020 18:34:15 +0300
Subject: [PATCH 1/2] padBuster.pl: remove unnecessary Crypt::SSLeay

Starting with version 6.02 of LWP, https support was unbundled into
LWP::Protocol::https. This module specifies as one of its prerequisites
IO::Socket::SSL which is automatically used by LWP::UserAgent unless this
preference is overridden separately. IO::Socket::SSL is a more complete
implementation, and, crucially, it allows hostname verification.
Crypt::SSLeay does not support this.
---
 padBuster.pl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/padBuster.pl b/padBuster.pl
index 95bd5bd..6ab6f25 100755
--- a/padBuster.pl
+++ b/padBuster.pl
@@ -20,7 +20,6 @@ use URI::Escape;
 use Getopt::Long;
 use Time::HiRes qw( gettimeofday );
 use Compress::Zlib;
-use Crypt::SSLeay;
 use File::Basename qw(dirname);
 use File::Path qw(make_path);
 
-- 
2.28.0

